dali/test/python/test_operator_reshape.py

```diff
@@ -207,13 +207,101 @@ def test_reshape_arg_input():
         for use_wildcard in [False, True]:
            yield check_reshape_with_arg_input, device, batch_size, relative, use_wildcard
 
+class ReinterpretPipeline1(Pipeline):
```

Comment:
a quick note of what is this (`ReinterpretPipeline1` and `ReinterpretPipeline2` doesn't tell me much) would help to read this code